torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2437,54 +2437,92 @@ Tensor linalg_qr_backward(const std::vector<torch::autograd::Variable> &grads, c
   }
 }
 
-// Invertible case is derived from Jacobi's formula, and also can be found at:
-// http://eprints.maths.ox.ac.uk/1079/1/NA-08-01.pdf
```

Comment:
Note, I removed this link because it's broken now. Added link to wikipedia page below